GhostMovement.md

```diff
@@ -0,0 +1,43 @@
+## Ghost movement logic
+
+Alternate Scatter 7 secs / chase 7 secs, duration changes as level increases.
+
+We need direction on the ghost object
+
+Ghosts only turn when at a junction (or corner)
+
+Ghosts reverse direction when switching mode.
+
+Ghosts are always moving towards a target square.
+
+* Is the ghost at a junction
+* Ghost checks the valid squares it can move to (can't move backwards)
+* Calculates which one is closest to target square
+* Turns in that direction
+
+
+### Scatter mode
+
+Head to their own home square.
+
+### Chase mode
+
+* Blinky - heads to Pacmans square
+* Pinky - heads to square 4 moves in front of Pacman (ignoring walls)
```

Comment:
Ignoring walls - I've watched a few people playing pacman on YouTube and they use this to their advantage...